.github/workflows/integration-test.yml

```diff
@@ -129,11 +129,3 @@ jobs:
           version: latest
           args: test:ui
           workdir: build
-
-      - name: Upload Playwright Report
-        uses: actions/upload-artifact@v3
-        if: ${{ always() }}
-        with:
-          name: playwright-report
-          path: playwright-report/
-          retention-days: 5
```

Comment:
Ah I see some playwright stuff reacting to the `CI` env var below.
Has this changed / is it using some kind of actions aware reporting?

If so, maybe this can be dropped: https://github.com/flipt-io/flipt/blob/d8a02eb55c4ce6aa56880eb12c91e7fe338ac3a6/build/testing/ui.go#L26-L30

Reply:
Unless you still want the pipeline to put this out locally. Then that makes sense and this can stay.

Reply:
Yeah, since the switch to Dagger, the UI ITs were not getting/respecting the `CI` env var that is automatically set by GH actions. This led to UI tests timing out (on failure) as it was trying to open a browser to display the HTML error results: https://github.com/flipt-io/flipt/actions/runs/5014436147/jobs/8988685030

So I just changed the playwright reporter to use a non-html one when running in CI, (which I think is the default anyways when you expose the `CI` env var). I'm not sure however what actually gets output instead of HTML report when running in CI though.. so maybe there is something in the `playwright-report` output that should be kept/uploaded? Will :eyes: